File: dotnetcore/StarterBot/Enums/State.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;

namespace StarterBot.Enums
{
    public enum State
    {
        READY,
        ACCELERATING,
        NOTHING,
        TURNING_RIGHT,
        TURNING_LEFT,
        HIT_MUD,
        HIT_OIL,
        DECELERATING,
        PICKED_UP_POWERUP,
        USED_BOOST,
        USED_OIL,
        FINISHED
    }
}

File: java/src/main/java/za/co/entelect/challenge/enums/State.java
package za.co.entelect.challenge.enums;

import com.google.gson.annotations.SerializedName;

public enum State {
    @SerializedName("ACCELERATING")
    ACCELERATING,
    @SerializedName("READY")
    READY,
    @SerializedName("NOTHING")
    NOTHING,
    @SerializedName("TURNING_RIGHT")
    TURNING_RIGHT,
    @SerializedName("TURNING_LEFT")
    TURNING_LEFT,
    @SerializedName("HIT_MUD")
    HIT_MUD,
    @SerializedName("HIT_OIL")
    HIT_OIL
}
